python/torch_mlir/dialects/torch/importer/jit_ir/csrc/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ include_directories(BEFORE
   )
 link_directories("${TORCH_INSTALL_PREFIX}/lib")
 
-add_library(TorchMLIRJITIRImporter MODULE
+add_library(TorchMLIRJITIRImporter SHARED
```

Comment:
It's set to `SHARED` since we need to link to `TorchMLIRJITIRImporter` from the lowering context, which isn't possible with `MODULE`.

More details available on this page: https://cmake.org/cmake/help/latest/command/add_library.html
> MODULE libraries are plugins that are not linked into other targets

Let me know if you wanted this embedded as a comment in the cmakefile itself

Reply:
Ah, ok, that's obvious enough. No need for a comment.